### DISCUSSION TITLE
can you add documentation on how to run webpack to generate the bundle.js for those who fork this library?

### DISCUSSION THREAD

**Discussion by simkimsia** [torchbox/django-pattern-library#201] [Ideas]
For those who want to fork this library and play with, can you include documentation on how to use webpack to generate the bundle.js?

and also how to experiment the forked version on a sandbox django project?

**Comment by thibaudcolas** [comment]
@simkimsia does this help? https://github.com/torchbox/django-pattern-library/blob/main/CONTRIBUTING.md This doesn’t mention Webpack because it’s largely an implementation detail but otherwise covers our front-end tooling.

> and also how to experiment the forked version on a sandbox django project?

I think this is a bit out of scope for our documentation as there is nothing special with this project compared to any other Django package in that respect – if you use `pip`, you’ll want to install your fork with `-e` ([editable mode](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs)).

If you just want a trial project for this library, there is one included within the repository. The simplest way to get to it is [with Gitpod](https://github.com/torchbox/django-pattern-library/blob/main/CONTRIBUTING.md#installation), but it’s also possible locally with Docker or Poetry.
